src/Json.Schema.ToDotNet/PropertyInfoDictionary.cs

```diff
@@ -630,7 +643,9 @@ private string GetUnqualifiedTypeName(string typeName, out string namespaceName)
                 namespaceName = null;
             }
 
-            return unqualifiedTypeName.ToPascalCase();
+            namespaceName = namespaceName == "System" ? null : namespaceName;
```

Comment:
>namespaceName = namespaceName == "System" ? null : namespaceName; [](http://example.com/codeflow?start=12&length=65)

This is because `using System` is not required.